Bottom: 90d806dffbfeb054adb0da15bcd1594ea194960d
Top:    a1de3d48c82964442a5250e0e214b1555ffbdbaa
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 22:09:27 +0100

hid/gtk: Enable transparent lines in GL renderer


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 0923110..5eb9851 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -79,7 +79,7 @@ ghid_set_layer (const char *name, int group, int empty)
 
   if (idx >= 0 && idx < max_copper_layer + 2)
     {
-      priv->trans_lines = false;
+      priv->trans_lines = true;
       return PCB->Data->Layer[idx].On;
     }
   if (idx < 0)
